openapi/userregistry.yaml

```diff
@@ -1,4 +1,26 @@
+swagger: '2.0'
 basePath: /pdnd-interop-uservice-user-registry-management/%7B%7Bversion%7D%7D
+schemes:
+  - https
+info:
+  contact:
+    email: support@example.com
+    name: API Support
+    url: 'http://www.example.com/support'
+  description: User Registry Management
+  termsOfService: 'http://swagger.io/terms/'
+  title: User Registry Management
+  version: '{{version}}'
+  x-api-id: an x-api-id
+  x-summary: an x-summary
+securityDefinitions:
+  SubscriptionKey:
+    type: apiKey
+    name: Ocp-Apim-Subscription-Key
+    in: header
+    description: The API key obtained through the developer portal.
```

Comment:
Change this description. I think that this key is not inside the developer portal.